src/parameterizations/vertical/MOM_bkgnd_mixing.F90

```diff
@@ -196,21 +224,56 @@ subroutine bkgnd_mixing_init(Time, G, GV, param_file, diag, CS)
 
   endif ! CS%Bryan_Lewis_diffusivity
 
+  call get_param(param_file, mdl, "HORIZ_VARYING_BACKGROUND", &
+                                CS%horiz_varying_background, &
+                 "If true, apply vertically uniform, latitude-dependent background\n"//&
+                 "diffusivity, as described in Danabasoglu et al., 2012", &
+                 default=.false.)
+
+  if (CS%horiz_varying_background) then
+    call check_bkgnd_scheme(CS, "HORIZ_VARYING_BACKGROUND")
+
+    call get_param(param_file, mdl, "BCKGRND_VDC1", &
+                   CS%bckgrnd_vdc1, &
+                   "Background diffusivity (Ledwell) when HORIZ_VARYING_BACKGROUND=True", &
+                   units="m2 s-1",default = 0.16e-04)
+
+    call get_param(param_file, mdl, "BCKGRND_VDC_EQ", &
+                   CS%bckgrnd_vdc_eq, &
+                   "Equatorial diffusivity (Gregg) when HORIZ_VARYING_BACKGROUND=True", &
+                   units="m2 s-1",default = 0.01e-04)
+
+    call get_param(param_file, mdl, "BCKGRND_VDC_PSIM", &
+                   CS%bckgrnd_vdc_psim, &
+                   "Max. PSI induced diffusivity (MacKinnon) when HORIZ_VARYING_BACKGROUND=True", &
+                   units="m2 s-1",default = 0.13e-4)
+
+    call get_param(param_file, mdl, "BCKGRND_VDC_BAN", &
+                   CS%bckgrnd_vdc_ban, &
+                   "Banda Sea diffusivity (Gordon) when HORIZ_VARYING_BACKGROUND=True", &
+                   units="m2 s-1",default = 1.0e-4)
+  endif
+
   call get_param(param_file, mdl, "HENYEY_IGW_BACKGROUND", &
                                 CS%Henyey_IGW_background, &
                  "If true, use a latitude-dependent scaling for the near \n"//&
                  "surface background diffusivity, as described in \n"//&
                  "Harrison & Hallberg, JPO 2008.", default=.false.)
+  if (CS%Henyey_IGW_background) call check_bkgnd_scheme(CS, "HENYEY_IGW_BACKGROUND")
+
 
   call get_param(param_file, mdl, "HENYEY_IGW_BACKGROUND_NEW", &
                                 CS%Henyey_IGW_background_new, &
                  "If true, use a better latitude-dependent scaling for the\n"//&
                  "background diffusivity, as described in \n"//&
                  "Harrison & Hallberg, JPO 2008.", default=.false.)
+  if (CS%Henyey_IGW_background_new) call check_bkgnd_scheme(CS, "HENYEY_IGW_BACKGROUND_NEW")
 
-  if (CS%Henyey_IGW_background .and. CS%Henyey_IGW_background_new) &
-     call MOM_error(FATAL, "set_diffusivity_init: HENYEY_IGW_BACKGROUND and \n"//&
-          "HENYEY_IGW_BACKGROUND_NEW are mutually exclusive. Set only one or none.")
+  if (CS%Kd>1.e-14 .and. (trim(CS%bkgnd_scheme_str)=="BRYAN_LEWIS_DIFFUSIVITY" .or.&
```

Comment:
This constant should be removed. It's has dimensions and fails the dimensional re-scaling test. Moreover, the test should be >0, not >small number.